isbn/conf.example.php

```diff
@@ -44,9 +44,9 @@
 //HEBIS = Hessisches BibliotheksInformationsSystem
 //http://www.hebis.de/de/1service/z3950/z39_zugang_index.php
 define('HEBIS_URL',         'tolk.hebis.de:20211/hebis');
-define('HEBIS_USER',        '3950');
+define('HEBIS_USER',        '');//TODO look up
 define('HEBIS_PASSWORD',    '');//TODO look up
-define('HEBIS_SYNTAX',      '');//TODO look up
+define('HEBIS_SYNTAX',      'marc21');
```

Comment:
http://www.hebis.de/de/1service/z3950/z39_zugang_index.php documents username / password. Is that combination wrong, or can it be used here?